_includes/primary-nav-toggle.html

```diff
@@ -0,0 +1,15 @@
+<button type="button" class="nav-btn c-primary-nav__item">
+  <span class="visuallyhidden">
+    Primary Navigation Toggle
+  </span>
+  <span class="toggle-open">
+    <svg class="c-icon c-primary-nav__icon" aria-hidden="true">
```

Comment:
Great to see someone using ARIA!

Just some suggestions:
1. Add `aria-controls` to help accessibility tools („AT”) understand the relationship between the button and the element it affects.
2. Add a `<title>` as first child of `<svg>` (+ aria-label), so AT can read out, what is displayed on the icon (assuming it is not used ornamentally).